rtp-session.go

```diff
@@ -92,8 +92,8 @@ func (s *RTPSession) DecodeRTCP(packetData []byte) (*RTCPCompoundPacket, error)
 		return nil, err
 	}
 
-	s.rtcpKey, s.rtcpSalt, err = s.kdf.DeriveForSRTCPStream(s.cipher,
-																													p.GetSRTCPIndex());
+	s.rtcpKey, s.rtcpSalt, err = s.kdf.DeriveForStream(s.cipher,
```

Comment:
There's no reason to re-derive the RTCP keys every time you decode a packet!  And you consolidated too far -- now the RTCP keys are being derived with the RTP labels.  Just extend `DeriveForStream` so that it returns four values instead of two, `DeriveForSteram(cipher CipherID) (rtpKey, rtpSalt, rtcpKey, rtcpSalt []byte, err error)`

Reply:
But I need the SRTCP index for the SRTCP key derivation.

Reply:
" now the RTCP keys are being derived with the RTP labels", missed that, will fix it

Reply:
Well this fix would result in reversing this fix commit...

Reply:
And yes there is a need to recalculate the key for every package:
https://tools.ietf.org/html/rfc3711#section-4.3.2
and
https://tools.ietf.org/html/rfc3711#section-3.4:
"The SRTCP index is a 31-bit counter for the SRTCP packet.
            The index is explicitly included in each packet, in contrast
            to the "implicit" index approach used for SRTP.  The SRTCP
            index MUST be set to zero before the first SRTCP packet is
            sent, and MUST be incremented by one, modulo 2^31, after
            each SRTCP packet is sent.  In particular, after a re-key,
            the SRTCP index MUST NOT be reset to zero again."

Reply:
No, just set the index to zero.  The spec says that, but if you look at what happens in practice, e.g., in libsrtp, the key is computed once, with the index equal to zero.  This is because the index only factors into the key derivation as `r = index DIV key_derivation_rate`, and in practice, `key_derivation_rate` is so high that this value never changes from zero.